value.go

```diff
@@ -59,11 +61,16 @@ func valueOfPointerOrInterface(v reflect.Value) js.Value {
 	return valueOf(v.Elem())
 }
 
-// valueOfSliceOrArray returns a new array object value.
-func valueOfSliceOrArray(v reflect.Value) js.Value {
+// valueOfSlice returns a new array object value.
+func valueOfSlice(v reflect.Value) js.Value {
 	if v.IsNil() {
```

Comment:
I created a separate function because calling `v.IsNil` on an array value results in a panic. We only need to make that check for slice values, but other than that the behavior is identical.

Reply:
Awesome, nice idea to call the new method.